src/main/resources/application.properties

```diff
@@ -4,6 +4,8 @@ spring.datasource.username=springApp
 spring.datasource.password=12345
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 #spring.jpa.show-sql: true
-
-
+spring.security.oauth2.client.registration.google.client-id=944892395889-f1ipq1mgfkh47nq70651u2nj5kn74rlj.apps.googleusercontent.com
+spring.security.oauth2.client.registration.google.client-secret=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
+spring.security.oauth2.client.registration.google.redirect-uri=http://localhost:8080/login/oauth2/code/google
```

Comment:
change to env variable
and remove from git history =)

Reply:
done